server/pulp/server/managers/repo/unit_association_query.py

```diff
@@ -411,10 +411,12 @@ def _associated_units_by_type_cursor(unit_type_id, criteria, associated_unit_ids
         sort = criteria.unit_sort
 
         if sort is None:
-            unit_key = types_db.type_units_unit_key(unit_type_id)
+            # dgregor - HACK - sort by _id so we use a single index
+            sort = [('_id', SORT_ASCENDING)]
 
-            if unit_key is not None:
-                sort = [(u, SORT_ASCENDING) for u in unit_key]
+            #unit_key = types_db.type_units_unit_key(unit_type_id)
+            #if unit_key is not None:
```

Comment:
These two lines are causing the Jenkins failure, since they violate PEP-8. We should just remove them, because we have a policy of not having commented-out code.